homeassistant/components/device_tracker/unifi.py

```diff
@@ -82,7 +90,8 @@ def _update(self):
             _LOGGER.error("Failed to scan clients: %s", ex)
             clients = []
 
```

Comment:
Porbably need something more like:
`self._clients = {client['mac']: client for client in clients
                 if (dt_util.utcnow() - dt_util.utc_from_timestamp(float(
                     client['last_seen']))) < self._detection_time}`

So that _clients is set with only devices that have talked to the unifi within detection_time